apps/api/src/app/notification-groups/usecases/update-notification-group/update-notification-group.command.ts

```diff
@@ -0,0 +1,10 @@
+import { IsString } from 'class-validator';
+import { EnvironmentWithUserCommand } from '../../../shared/commands/project.command';
+
+export class UpdateNotificationGroupCommand extends EnvironmentWithUserCommand {
+  @IsString()
```

Comment:
This param should be mandatory. I am afraid of the side effects of it as not sure if not having an id could potentially make the repository to update all entries with the same name. That's something we can check with a test to know if we would need to have more safety barriers.